packages/rxjs/spec/operators/share-spec.ts

```diff
@@ -856,89 +855,86 @@ describe('share', () => {
+        expect(err).to.equal(error);
+        done();
       });
-    });
 
-    it('should not call "resetOnRefCountZero" on error', () => {
       rxTest.run(({ cold, expectObservable, expectSubscriptions }) => {
-        const resetOnRefCountZero = spy(() => EMPTY);
-
-        const source = cold('    ---1---(2#)                ');
-        // source: '                           ---1---(2#)  '
-        const sourceSubs = [
-          '                      ^------(! )                ',
-          // break the line, please
-          '                      -------(- )---^------(! )  ',
-        ];
-        const expected = '       ---1---(2 )------1---(2#)  ';
-        const subscription = '   ^------(- )----------(- )  ';
-        const firstPause = cold('       (- )---|            ');
-        const reset = cold('            (- )-r              ');
-        // reset: '                                   (- )-r'
+        const source = cold('    ---1---2---|   ');
+        const sourceSubs = '     ^----------!   ';
+        const expected = '       ---1---2------|';
+        const subscription = '   ^--------------';
+        const firstPause = cold('       -------|');
+        const reset = cold('                --# ', undefined, error);
 
-        const sharedSource = source.pipe(share({ resetOnError: () => reset, resetOnRefCountZero }));
+        const sharedSource = source.pipe(share({ resetOnComplete: () => reset, resetOnRefCountZero: false }), take(2));
 
-        const result = concat(sharedSource.pipe(onErrorResumeNextWith(firstPause)), sharedSource);
+        const result = concat(sharedSource, firstPause, sharedSource);
 
         expectObservable(result, subscription).toBe(expected);
         expectSubscriptions(source.subscriptions).toBe(sourceSubs);
-        expect(resetOnRefCountZero).to.not.have.been.called;
       });
     });
+  });
 
-    it('should not call "resetOnRefCountZero" on complete', () => {
-      rxTest.run(({ cold, expectObservable, expectSubscriptions }) => {
-        const resetOnRefCountZero = spy(() => EMPTY);
-
-        const source = cold('    ---1---(2|)                ');
-        // source: '                           ---1---(2|)  '
-        const sourceSubs = [
-          '                      ^------(! )                ',
-          // break the line, please
-          '                      -------(- )---^------(! )  ',
-        ];
-        const expected = '       ---1---(2 )------1---(2|)  ';
-        const subscription = '   ^------(- )----------(- )  ';
-        const firstPause = cold('       (- )---|            ');
-        const reset = cold('            (- )-r              ');
-        // reset: '                                   (- )-r'
-
-        const sharedSource = source.pipe(share({ resetOnComplete: () => reset, resetOnRefCountZero }));
-
-        const result = concat(sharedSource, firstPause, sharedSource);
+  it('should not call "resetOnRefCountZero" on error', () => {
```

Comment:
Everything under this section is really a whitespace change. I'm not sure why the diff is flipping out here. This was just from reorganizing things in the file I think.